packages/bpk-component-calendar/src/BpkCalendarGridTransition.js

```diff
@@ -147,7 +147,11 @@ class BpkCalendarGridTransition extends Component {
       isTransitioning && 'bpk-calendar-grid-transition__strip--transitioning',
     );
 
-    const { min, max } = getMonthRange(rest.minDate, rest.maxDate);
+    let min;
+    let max;
```

Comment:
I think this could be changed to 1 line but not a biggie
```suggestion
    let min, max;
```

Reply:
Nice idea

Reply:
lol this broke CI – eslint forbids this. I'll revert